utils/tagrecommendation_utilities.py

```diff
@@ -50,21 +51,15 @@ def get_recommended_tags(input_tags, max_number_of_tags=30):
             recommended_tags = False
 
     if not recommended_tags:
-        try:
-            recommended_tags = TagRecommendation.recommend_tags(input_tags)
-
-            if not recommended_tags['tags']:
-                recommended_tags['community'] = "-"
+        recommended_tags = TagRecommendation.recommend_tags(input_tags)
 
-            try:
-                cache.set(cache_key, recommended_tags, TAGRECOMMENDATION_CACHE_TIME)
-            except:
-                pass
+        if not recommended_tags['tags']:
+            recommended_tags['community'] = "-"
 
-        except Exception, e:
-            logger.error('Could not get a response from the tagrecommendation service (%s)\n\t%s' % \
-                         (e, traceback.format_exc()))
-            recommended_tags = False
+        try:
+            cache.set(cache_key, recommended_tags, TAGRECOMMENDATION_CACHE_TIME)
+        except:
```

Comment:
Even though this was just copied from above, it would be nice to take the chance to fix exceptions here as well, since you're working on this code